R/setup_source.R

```diff
@@ -0,0 +1,73 @@
+#' Add or update the checklist infrastructure to a repository with source files.
```

Comment:
As `create_source()` does not exist, and users often do not know how such a repo should be organised (I got a question on it last week...), it would be interesting to refer to information on this subject (or some publicly available examples).  Or if this is not available, it would be helpful to write a tutorial/vignet on the subject: 'how to create an R project for analyzing and reporting scientific data'.

Reply:
Good idea. I'm trying to wrap up this PR, so it is something for the future. Can you create an issue? Or even better a PR?

Reply:
I copied it to issue #41.  For the moment I unfortunately don't have time to work on a PR, but I may pick up it later.